components/calendar/src/iso.rs

```diff
@@ -395,6 +397,7 @@ impl Iso {
     }
 
     fn fixed_from_iso_integers(year: i32, month: i32, day: i32) -> i32 {
+        #[allow(clippy::unwrap_used)] // TODO(#NNN) Clippy exceptions need docs or fixing.
```

Comment:
Suggestion: For cases like this, please take the time to investigate the safety of the unwrap. In this case, I see that this is a private function with 3 call sites. Try to structure the code such that the clippy exception is closer to the call site where it is quickly obvious that there is no risk of panic.